gulpTasks/definition.js

```diff
@@ -54,7 +54,7 @@ function clearEnumVariableDeclaration(match, p1, p2) {
 function getEnumRegex() {
   const enumIdentifier = '(enum [a-zA-Z_$]+\\s{$)';
   const enumDeclaration = '\\n^\\s*[a-zA-Z_$]+ = "[a-zA-Z_$\\s]+",?$';
-  const documentation = '\\n^\\s*[@{}\\[\\]\\w\\/*.,\\s]+$';
+  const documentation = '\\n^\\s*[@{}\\[\\]\\w\\/*.,\\s\\(\\)\\|;]+$';
```

Comment:
Why is this change needed?

Reply:
Because `validateTSV1` doesn't support enums with string keys unless the `= 'string value'` part is removed. This was fixed for other enums with this function, but the newly introduced [`FacetSearchType`](https://github.com/coveo/search-ui/blob/master/src/rest/Facet/FacetSearchRequest.ts#L6) has special symbols in its comments that weren't caught in the regex.